pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -50,16 +53,26 @@ def add_arguments(self, parser):
                  "are excluded.",
         )
 
-    def _get_revision_from_since(self, since):
-        from pootle.core.dateparse import parse_datetime
-        from pootle.core.utils.timezone import make_aware
-        from pootle_statistics.models import Submission
+    def get_aware_datetime(self, date_string):
```

Comment:
now that we are just using dateutil and the existing make_aware function - im not sure that there is any benefit to separating out this method or adding tests for it